lib/core_extensions.rb

```diff
@@ -90,3 +90,15 @@ def are_all_conflicts?
     self[:conflict].count + self[:'interfaces.conflict'].count == count
   end
 end
+
+module ActiveModel
+  module Attributes
+    def write_attribute(attr_name, value)
```

Comment:
Given just ActiveModel is affected, there might be a way to update them.
Could we add this with Deprecation warning and update the occurences in core?